src/test/java/teammates/test/cases/util/FieldValidatorTest.java

```diff
@@ -1,19 +1,19 @@
 package teammates.test.cases.util;
 
 // CHECKSTYLE.OFF:AvoidStarImport as we want to perform tests on everything from FieldValidator
-import static teammates.common.util.FieldValidator.*;
-// CHECKSTYLE.ON:AvoidStarImport
+import static teammates.test.driver.FieldValidator.*;
 
 import java.util.Date;
 
 import org.testng.annotations.Test;
 
 import com.google.appengine.api.datastore.Text;
 
-import teammates.common.util.FieldValidator;
 import teammates.common.util.StringHelper;
-import teammates.common.util.TimeHelper;
 import teammates.test.cases.BaseTestCase;
+import teammates.test.driver.FieldValidator;
```

Comment:
Now that @damithc has weighed in that we should not test through a subclass, use the `Extension` suffix here as well. The extension can be a util class (final + private constructor) and the method static.